README.md

```diff
@@ -91,9 +91,9 @@ For either request (HTTP 400) or response (HTTP 500) errors, details of the sche
 }
 ```
 
-### `ui(document) => koa2 middleware`
+### `ui(document, basePath = "/", skipPaths = []) => koa2 middleware`
 
-You can also serve a swagger-ui for your API:
+You can also serve a swagger-ui for your API from a given base path (basePath defaults to "/"):
```

Comment:
Can you rename basePath to something like pathRoot (as below).  Using the same term as the swagger spec (basePath) makes me think this has something to do with the API definition itself.